easybuild/easyconfigs/l/LAMMPS/LAMMPS-7Aug2019-intel-2019b-Python-3.7.4-kokkos-OCTP.eb

```diff
@@ -0,0 +1,112 @@
+name = 'LAMMPS'
+version = '7Aug2019'
+local_python_versionsuffix = '-Python-%(pyver)s'
+versionsuffix = local_python_versionsuffix
+versionsuffix += '-kokkos-OCTP'
+
+homepage = 'https://lammps.sandia.gov/'
+description = """LAMMPS is a classical molecular dynamics code, and an acronym
```

Comment:
I think the description should also contain something on OCTP. I for one have no idea what it is, and what would make this installation different from a standard LAMMPS.

Reply:
done in 07d1e28